Source/Controllers/MainViewControllers/SPExtendedTableInfo.m

```diff
@@ -711,24 +711,32 @@ - (NSString *)_formatValueWithKey:(NSString *)key inDictionary:(NSDictionary *)i
 				 [key isEqualToString:SPMySQLUpdateTimeField]) {
 
 			// Create date formatter
-			NSDateFormatter *dateFormatter = [[[NSDateFormatter alloc] init] autorelease];
-
-			[dateFormatter setFormatterBehavior:NSDateFormatterBehavior10_4];
-
-			[dateFormatter setDateStyle:NSDateFormatterMediumStyle];
-			[dateFormatter setTimeStyle:NSDateFormatterMediumStyle];
-
+			// Create date formatter - once
+			static NSDateFormatter *dateFormatter = nil;
```

Comment:
Can we create `DateFormatterExtension.swift` file where we will have `@objc static var mediumStyleFormatter: NSDateFormatter` and we will reuse this across the app + it will be unified on one place in swift?

The same for Number formatter etc?

It's way better to have one place for one and the same thing. also it will be reused across controllers and classes :)

Reply:
so in obj c we use:` [NSDateFormatter.mediumStyleFormatter method:]`?

Reply:
@Kaspik ah we have other date and time styles. So, multiple static vars, or a func that takes the styles>?

Reply:
Definitely multiple static vars :)

Reply:
Didn't need it in the end